Dockerfile

```diff
@@ -1,16 +1,23 @@
-FROM alpine:3.17 AS base
+FROM golang:1.20 AS build
 
 WORKDIR /app
 
-FROM base AS builder
+# Make sure the dependency downloading can be cached
+COPY go.mod go.sum ./
+RUN go mod download
 
 COPY . .
+RUN CGO_ENABLED=0 GOOS=linux go build -o ./vulnbot
 
-RUN apk update && apk upgrade && apk add go
-RUN go build .
+FROM build as test
```

Comment:
[suggestion] add a brief comment for this `test` layer to clarify its intent / scope.

It's obvious now with PR context but might become less obvious as the image evolves.